marklogic-data-hub/src/main/resources/ml-modules/root/com.marklogic.hub/mlcp-flow-transform.xqy

```diff
@@ -78,7 +78,8 @@ declare function mlcpFlow:transform(
 declare function mlcpFlow:run-flow(
   $jobId, $flow, $uri, $content, $options)
 {
-  (: mlcp in runs in update mode :)
+  try{
```

Comment:
the old code would not catch an exception and the error would bubble up to mlcp. Why the try/catch here?

Reply:
I think mlcp was showing a dummy line like "there was an error" but the stack/details were lost, so I added that while debugging an issue, and left it in as useful.